types/rower.model.ts

```diff
@@ -0,0 +1,25 @@
+export interface Rower {
+	id: string;
+	regatta_id: string;
+	club_id: string;
+	knrb: string;
+	knrb_num: null; // Is dit hetzelfde als 'knrb'?
+	license: number;
+	fullName: string;
+	initals: string;
+	firstname: string;
+	middlename: string;
+	lastname: string;
+	year_of_birth: null;
+	gender: string;
+	_joinData: JoinData; // Wat is JoinData?
```

Comment:
Dat is cakephp eigen

Reply:
Hier in word informatie van de jointable toegevoegd

Reply:
Dus dat is hier CrewMember of iets anders

Reply:
Is dat handig om te hebben voor de frontend of zal ik het weglaten?

Reply:
Of kan dat niet gewoon worden samengevoegd door cake?

Reply:
Zal ik naar kijken